examples/contrib/cifar10/main.py

```diff
@@ -187,6 +187,13 @@ def run(
         **spawn_kwargs: Other kwargs to spawn run in child processes: master_addr, master_port, node_rank, nnodes
 
     """
+    # check to see if the num_epochs is greater than num_warmup_epochs issue user friendly Error
+    if num_warmup_epochs > num_epochs:
+        raise ValueError(
+            f"The minimum number of warmup epochs is {num_warmup_epochs}, either increase the num_epochs or decrease"
```

Comment:
The error message is not understandable for me, it should be the minimum number of (epochs) must be greater than or equal to num_warmup_epochs, am I missing something?
To be more clear I suggest to do something like
```
epochs can't be less than num_warmup_epochs, etc....
```

Reply:
I won't spam and put a lot of comments, this also the case of other updates. Thanks!

Reply:
Right! Sorry for that, will make the changes soon enough and commit. Thanks for the inputs!